modules/admin_manual/pages/troubleshooting/providing_logs_and_config_files.adoc

```diff
@@ -0,0 +1,34 @@
+= Providing log files and configurations
```

Comment:
Please use camel case for all headers

Reply:
[Title case](https://www.thoughtco.com/title-case-capitalization-1692469) is a more universal term.